src/rtctools_heat_network/esdl/esdl_heat_model.py

```diff
@@ -645,6 +645,7 @@ def convert_heat_pump(self, asset: Asset) -> Tuple[Type[HeatPump], MODIFIERS]:
 
         modifiers = dict(
             COP=cop,
+            efficiency=asset.attributes["efficiency"] if asset.attributes["efficiency"] else 1.0,
```

Comment:
default value of the efficiency should actually be lower than 1.0 to represent reality. Typically the efficiency would be 50-80%. Setting it to 0.8 still results in a optimisitic but somewhat more realistic efficiency.

Reply:
I agree, I will actually set it to 50%